packages/api/src/models/Expense.ts

```diff
@@ -15,39 +15,38 @@ export enum ExpenseStatus {
 }
 
 export default interface Expense {
-	categoryId: string
+	categoryId: number
 	markupPercent?: number
-	projectId?: string
-	clientId?: string
-	taxPercent1?: Nullable<number>
+	projectId?: number
+	clientId?: number
+	taxPercent1?: Nullable<string>
 	taxName2?: Nullable<string>
 	taxName1?: Nullable<string>
 	isDuplicate?: boolean
-	profileId?: Nullable<string>
-	taxPercent2?: Nullable<number>
+	profileId?: Nullable<number>
+	taxPercent2?: Nullable<string>
 	accountName?: string
-	transactionId?: Nullable<string>
-	invoiceId?: Nullable<string>
-	id?: string
+	transactionId?: Nullable<number>
+	invoiceId?: Nullable<number>
+	id?: number
 	taxAmount2?: Nullable<Money>
 	taxAmount1?: Nullable<Money>
 	visState?: VisState
 	status?: ExpenseStatus
 	bankName?: string
 	updated?: Date
 	vendor?: Nullable<string>
-	extSystemId?: string
-	staffId: string
+	extSystemId?: number
+	staffId: number
 	date: Date
 	hasReceipt?: boolean
 	accountingSystemId?: string
-	backgroundJobId?: Nullable<string>
 	notes?: string
-	extInvoiceId?: string
-	amount: Money
-	expenseId?: string
+	extInvoiceId?: number
+	amount: Money // There are two amounts in the API - one is a string and other an object.
```

Comment:
The string is within the object, is it not?

Reply:
Ahh I see - the code and amount (string) are subfields.